js/annotations/annotations.src.js

```diff
@@ -238,6 +239,8 @@ merge(
              * Options for annotation's labels. Each label inherits options
              * from the labelOptions object. An option from the labelOptions
              * can be overwritten by config for a specific label.
+             *
+             * @requires modules/annotations
```

Comment:
If the `labelOptions` have a `@required` tag, its brother `shapeOptions` should also have it.

Reply:
Please review once more.